src/extension.ts

```diff
@@ -1000,7 +1000,10 @@ function appendSuffix(label: string): string {
         if (split[i] === "JCL" || split[i] === "CNTL" ) {
             return label.concat(".jcl");
         }
-        if (split[i] === "COBOL" || split[i] === "CBL" ) {
+        if (split[i] === "COBOL" || split[i] === "CBL" || split[i] === "COB" ) {
+            return label.concat(".cbl");
+        }
+        if (["COPYBOOK", "COPY", "COBCOPY", "CPY"].includes(split[i])) {
             return label.concat(".cbl");
```

Comment:
@Colin-Stone  Should this be `.cpy` instead of `.cbl` ??

Reply:
> @Colin-Stone Should this be `.cpy` instead of `.cbl` ??

Excellent you passed my little test ;-)